### DISCUSSION TITLE
Migration failed - no env. file found

### DISCUSSION THREAD

**Discussion by MRChriz385** [dani-garcia/vaultwarden#3467] [Q&A]
Hi guys, 

i am relatively new to docker & vaultwarden. I now have the problem that after an attempted migration of bitwarden_rs the bitwarden is no longer accessible. It is hosted on a Synology NAS via a Docker image. 

I need the access very urgently, because important passwords are used for work. 

Is there any way to make the copied data (from Docker image to NAS volume) available again ? 

Thanks a lot for your help!

**Comment by MRChriz385** [comment]
Any idea?

**Comment by BlackDex** [comment]
Try to read the following pages
https://github.com/dani-garcia/vaultwarden/wiki/General-%28not-docker%29

https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault
